store/store.go

```diff
@@ -93,13 +94,20 @@ func NewStore(conf *Config) (Store, error) {
 		txStore:        newTxStore(db, conf.TxCacheSize),
 		accountStore:   newAccountStore(db, conf.AccountCacheSize),
 		validatorStore: newValidatorStore(db),
+		isPruned:       false,
```

Comment:
Don't require to set the default value, because the default value boolean is false

Reply:
It's okay. Let it be. Some linters are sensitive about not setting values explicitly.